requirements.txt

```diff
@@ -3,6 +3,7 @@ torchvision==0.11.*
 torchtext==0.11.*
 fairscale==0.4.*
 pytorch-lightning==1.5.*
+jsonargparse[signatures]
```

Comment:
Shouldn't we make the very same change in `environment.yml` too (until #4 is resolved)?

Reply:
... and maybe also constrain the version to `jsonargparse[signatures]==4.0.*`?

Reply:
yep